test/unit/memory/store/tuple/aligned.cpp

```diff
@@ -0,0 +1,66 @@
+//==================================================================================================
+/**
+  EVE - Expressive Vector Engine
+  Copyright : EVE Contributors & Maintainers
+  SPDX-License-Identifier: MIT
+**/
+//==================================================================================================
+#include "test.hpp"
+#include <eve/memory/aligned_ptr.hpp>
+#include <eve/function/store.hpp>
+
+#include <array>
+#include <numeric>
+
+namespace
+{
+
+template<typename T>
+using tuple_t = kumi::tuple<std::int8_t,T,double>;
+
+//==================================================================================================
+// Aligned store tests
+//==================================================================================================
+EVE_TEST_TYPES( "Check store behavior with aligned pointers", eve::test::scalar::all_types)
+<typename T>(eve::as_<T>)
+{
+  using s_t  = tuple_t<T>;
+  using w_t = eve::wide<tuple_t<T>>;
+
+  auto const filler = [](auto i, auto)  { return s_t{ static_cast<std::int8_t>('a'+i)
+                                                    , static_cast<T>(i + 1)
+                                                    , 1.5*(1+i)
+                                                    };
+                                        };
+
+  w_t data = filler;
+
+  std::array<std::int8_t, w_t::size()> ref0;
+  std::array<T          , w_t::size()> ref1;
+  std::array<double     , w_t::size()> ref2;
+
+  for(int i = 0; i < w_t::size(); ++i)
+  {
+    ref0[i] = static_cast<std::int8_t>('a'+i);
+    ref1[i] = static_cast<T>(i + 1);
+    ref2[i] = 1.5*(1+i);
+  }
+
+  constexpr auto algt = eve::alignment_v<w_t>;
+  alignas(algt) std::array<std::int8_t, w_t::size()> target0;
+  alignas(algt) std::array<T          , w_t::size()> target1;
+  alignas(algt) std::array<double     , w_t::size()> target2;
```

Comment:
Yeah, I have a problem with 1 alignment value in bytes

Reply:
Will as fix as per discussion